### ISSUE TITLE
Saving

### ISSUE THREAD

**Issue by rom1504** [mhsjlw/minecraft-classic-anvil#2]
https://www.npmjs.com/package/java.io#2-writeobjectobj

**Comment by mhsjlw** [comment]
yeah, I am going to add a buffer field for now, so you pretty much give it a buffer and it replaces it in the object. eventually though, I can add world.js functionality inside of minecraft-classic-anvil.
